mavros/src/plugins/sys_status.cpp

```diff
@@ -430,16 +431,23 @@ class SystemStatusPlugin : public plugin::PluginBase
 		double conn_timeout_d;
 		double conn_heartbeat_d;
 		double min_voltage;
+		std::string conn_heartbeat_mav_type_str;
+		
 
 		nh.param("conn/timeout", conn_timeout_d, 10.0);
 		nh.param("sys/min_voltage", min_voltage, 10.0);
 		nh.param("sys/disable_diag", disable_diag, false);
 
-		// rate parameter
+		// heartbeat rate parameter
 		if (nh.getParam("conn/heartbeat_rate", conn_heartbeat_d) && conn_heartbeat_d != 0.0) {
 			conn_heartbeat = ros::Duration(ros::Rate(conn_heartbeat_d));
 		}
 
+		// heartbeat mav type parameter
+		if (nh.getParam("conn/heartbeat_mav_type", conn_heartbeat_mav_type_str) && !conn_heartbeat_mav_type_str.empty()) {
```

Comment:
`.empty()` check is not necessary. Heartbeat rate check is needed because we can set 0 to disable heartbeats.